attribute.go

```diff
@@ -22,7 +22,7 @@ func extractAttribute(a *Attribute, logger Logger, data []byte) error {
 		case nfUlaAttrPacketHdr:
 			hwProtocol := binary.BigEndian.Uint16(ad.Bytes()[:2])
 			a.HwProtocol = &hwProtocol
-			hook := uint8(ad.Bytes()[3])
+			hook := uint8(ad.Bytes()[2])
```

Comment:
Just checked the backing structure on the Kernel side.
https://github.com/torvalds/linux/blob/fac04efc5c793dccbd07e2d59af9f90b7fc0dca4/include/uapi/linux/netfilter/nfnetlink_log.h#L19-L23

Not sure why this indexing bug hasn't been noticed so far. Thanks for idenfying and fixing it!

Can you also update the tests, then I will merge your fix.

Reply:
No worries. Thanks for the library!

I have updated the tests. Instead of checking for `0x00` they are now checking for `0x03` and 0x01` respectively, which is maybe what you intended originally? Testing for the zero value seems unsound.

The solution proposed originally is fine? You are sure this has nothing to do with endianess?